awscli/customizations/cloudformation/deploy.py

```diff
@@ -169,6 +169,30 @@ class DeployCommand(BasicCommand):
                 ' execute it to implement your changes.'
             )
         },
+        {
+            'name': 'disable-rollback',
```

Comment:
I didn't see info on what happens if we pass both `--disable-rollback` and `--no-disable-rollback` in the same command. What's returned to the end user in that case?

Reply:
I tested with create/update stack api, seems like if user pass both flag, the second flag will override first flag and control disable-rollback flag. For example, if we do
```
aws cfn create-stack ... --disable-rollback --no-disable-rollback
```
The flag `no-disable-rollback` will override `disable-rollback` so this flag will become false

@atheiman Would u mind also help verify this locally for your corresponding change?

Reply:
Im not sure what the functionality _should_ be if a user specifies `--disable-rollback` and `--no-disable-rollback`, but the **currently the second specified argument takes precendence** in my local testing:

```
# --disable-rollback --no-disable-rollback => stack is rolled back
(.venv) aws-cli $ aws cloudformation deploy --template-file ~/tmp/cfn-error.yml --stack-name rollbacktest --disable-rollback --no-disable-rollback

Waiting for changeset to be created..
Waiting for stack create/update to complete

Failed to create/update the stack. Run the following command
to fetch the list of events leading up to the failure
aws cloudformation describe-stack-events --stack-name rollbacktest
(.venv) aws-cli $ aws cloudformation describe-stack-events --stack-name rollbacktest --output text --query 'StackEvents[][Timestamp, LogicalResourceId, ResourceStatus]'
2021-11-11T23:01:21.235Z	rollbacktest	ROLLBACK_COMPLETE
2021-11-11T23:01:20.405Z	BadSsmParameter	DELETE_COMPLETE
2021-11-11T23:01:11.466Z	rollbacktest	ROLLBACK_IN_PROGRESS
2021-11-11T23:01:10.643Z	BadSsmParameter	CREATE_FAILED
2021-11-11T23:01:08.132Z	BadSsmParameter	CREATE_IN_PROGRESS
2021-11-11T23:01:02.282Z	rollbacktest	CREATE_IN_PROGRESS
2021-11-11T23:00:56.605Z	rollbacktest	REVIEW_IN_PROGRESS
```

If the parameters are reversed on the command line the result is the opposite

```
# --no-disable-rollback --disable-rollback => stack is not rolled back
(.venv) aws-cli $ aws cloudformation deploy --template-file ~/tmp/cfn-error.yml --stack-name rollbacktest --no-disable-rollback --disable-rollback

Waiting for changeset to be created..
Waiting for stack create/update to complete

Failed to create/update the stack. Run the following command
to fetch the list of events leading up to the failure
aws cloudformation describe-stack-events --stack-name rollbacktest
(.venv) aws-cli $ aws cloudformation describe-stack-events --stack-name rollbacktest --output text --query 'StackEvents[][Timestamp, LogicalResourceId, ResourceStatus]'
2021-11-11T23:02:58.795Z	rollbacktest	CREATE_FAILED
2021-11-11T23:02:57.697Z	BadSsmParameter	CREATE_FAILED
2021-11-11T23:02:55.767Z	BadSsmParameter	CREATE_IN_PROGRESS
2021-11-11T23:02:51.585Z	rollbacktest	CREATE_IN_PROGRESS
2021-11-11T23:02:40.787Z	rollbacktest	REVIEW_IN_PROGRESS
```

@nateprewitt @SeveroeHe how should this be handled? I dont know what the user interface should be. I think the current implementation is _fine_ but just needs to be documented? Any suggestions on what the documentation should be?

Reply:
I think your test results look good as the behavior aligned with other CFN APIs that support disable-rollback flag